Sources/Xcodeproj/pbxproj().swift

```diff
@@ -582,6 +582,13 @@ func xcodeProject(
             // If user provided the modulemap no need to generate.
             if fileSystem.isFile(clangTarget.moduleMapPath) {
                 moduleMapPath = clangTarget.moduleMapPath
+                let headerPhase = xcodeTarget.addHeadersBuildPhase()
+                for case let header as Xcode.FileReference in includeGroup.subitems {
+                  let buildFile = headerPhase.addBuildFile(fileRef: header)
+                  buildFile.settings.ATTRIBUTES = ["Public"]
+                }
+                targetSettings.common.CLANG_ENABLE_MODULES = "YES"
+                targetSettings.common.DEFINES_MODULE = "YES"
```

Comment:
We purposely don't do this because it breaks indexing and code completion since we're trying to mix two models (framework and non-framework modules). The only case where this works is when you don't provide a module map and there is an umbrella header.

Reply:
I can see you point. I asked a Swift engineer in the lab and came up with the conclusion that I will close this PR and issue :)